test/golang/Dockerfile

```diff
@@ -43,11 +46,9 @@ FROM build as testb
 RUN install-tool golang 1.11.0
 
 RUN set -ex; \
-    cd d; \
-    go get -d ./...; \
-    go mod tidy;
-
-USER 1000
+  cd d; \
```

Comment:
What's causing those indention changes?

Reply:
My prettifyer... I'll change it back

Reply:
Do i missed to add a prettier config? 🤔

We should have it to suppress those changes in future. 🤗